core-ui/src/components/Lambdas/config.js

```diff
@@ -115,13 +115,23 @@ const defaultConfig = {
     cpu: '200m',
   },
   namespaceContainerLimitsPreset: {
+    S: { max: '550Mi', default: '250Mi', defaultRequest: '16Mi' },
```

Comment:
It's lambdas config, please move the namespace presets to the namespaces folder.